apps/fluent-tester/src/E2E/Switch/specs/Switch.spec.win.ts

```diff
@@ -0,0 +1,65 @@
+import NavigateAppPage from '../../common/NavigateAppPage.win';
+import SwitchPageObject from '../pages/SwitchPageObject';
+import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT, BUTTON_A11Y_ROLE } from '../../common/consts';
+
+// Before testing begins, allow up to 60 seconds for app to open
+describe('Switch Testing Initialization', function () {
+  it('Wait for app load', () => {
+    NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
+    expect(NavigateAppPage.isPageLoaded()).toBeTruthy(NavigateAppPage.ERRORMESSAGE_APPLOAD);
+  });
+
+  it('Click and navigate to Switch test page', () => {
+    /* Scroll to component test page button in scrollview if not already visible*/
+    SwitchPageObject.scrollToComponentButton();
+    SwitchPageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
+
+    /* Click on component button to navigate to test page */
+    NavigateAppPage.clickAndGoToSwitchPage();
+    SwitchPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
+
+    expect(SwitchPageObject.isPageLoaded()).toBeTruthy(SwitchPageObject.ERRORMESSAGE_PAGELOAD);
+    expect(SwitchPageObject.didAssertPopup()).toBeFalsy(SwitchPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+  });
+});
+
+describe('Switch Accessibility Testing', () => {
+  /* Scrolls and waits for the Switch to be visible on the Test Page */
+  beforeEach(() => {
+    SwitchPageObject.scrollToTestElement();
+    SwitchPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+  });
+
+  it('Switch - Validate accessibilityRole is correct', () => {
+    expect(SwitchPageObject.getAccessibilityRole()).toEqual(BUTTON_A11Y_ROLE);
```

Comment:
Is this correct? The switch is a Button? In the spec, it says accessibilityRole = 'switch'

Reply:
Yeah there isn't a Switch control type identifier. You can see that [here](https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-controltype-ids). It gets resolved as a button in win32 I believe.